app/views/school_groups/_enhanced_header.html.erb

```diff
@@ -1,4 +1,8 @@
-<% content_for :page_title, @school_group.name + ' | ' +  t("school_groups.titles.#{action_name}") %>
+<%= render 'sub_nav' if can?(:update_settings, @school_group) %>
+
+<p class='pt-4'>
```

Comment:
Do you need the p class here? The page title content isn't visible, so shouldn't need formatting. If you need a space then add to the breadcrumb class?